ta/pkcs11/src/processing_symm.c

```diff
@@ -149,34 +222,171 @@ allocate_tee_operation(struct pkcs11_session *session,
 	return tee2pkcs_error(res);
 }
 
+static enum pkcs11_rc pkcs2tee_load_hashed_attr(TEE_Attribute *tee_ref,
+						uint32_t tee_id,
+						struct pkcs11_object *obj,
+						enum pkcs11_attr_id pkcs11_id,
+						uint32_t tee_algo,
+						void *hash_ptr,
+						uint32_t *hash_size)
+{
+	TEE_OperationHandle handle = TEE_HANDLE_NULL;
+	void *a_ptr = NULL;
+	uint32_t a_size = 0;
+	TEE_Result res = TEE_ERROR_GENERIC;
+
+	if (get_attribute_ptr(obj->attributes, pkcs11_id, &a_ptr, &a_size))
+		return PKCS11_CKR_FUNCTION_FAILED;
+
+	/* Calculate hash of the attribute */
```

Comment:
can remove this inline comment, or maybe move it at line 225, as `pkcs2tee_load_hashed_attr()` description inline comment.

Reply:
Sure. I will also move this function to file pkcs11_helpers.c